sound/soc/sof/topology.c

```diff
@@ -2019,6 +2042,108 @@ static int sof_get_control_data(struct snd_soc_component *scomp,
 	return 0;
 }
 
+/*
+ * Sof mux/demux component
+ */
+static int sof_process_load_mux(struct snd_soc_component *scomp,
+				struct sof_ipc_comp_process *process,
+				struct snd_soc_tplg_private *private,
+				int *offset)
+{
+	struct sof_ipc_stream_map *sm =
+		(struct sof_ipc_stream_map *)((u8 *)process->data + (*offset));
+	u8 *pos = (u8 *)process->data + (*offset) +
+		sizeof(struct sof_ipc_stream_map);
```

Comment:
sizeof(*sm)?

Reply:
I'm not sure about the customs with sizeof, but sure can change

Reply:
ok